config/webhook-cert/job.yaml

```diff
@@ -3,6 +3,8 @@ kind: Job
 metadata:
   name: webhook-cert-init
   namespace: system
+  annotations:
+    argocd.argoproj.io/sync-options: Force=true,Replace=true
```

Comment:
We could also add this annotation during bootstrap (preprocessManifests). This would have the advantage that we can only add it if "gitops" mode is enabled. What do you think?

Reply:
Yes it's a good idea, will do.

Reply:
Ok, not so sure anymore. Now if somebody updates with the dry-run bootstrap, it doesn't know whether it is in gitops mode or not? So it would remove the annotation.

Reply:
We could store this information as a setting via namespace annotation

Reply:
Thanks, yes we already do that but I forgot about it